### ISSUE TITLE
Add CMakeLists.txt for Samples

### ISSUE THREAD

**Issue by chatchoi** [DavidNash2024/Win32xx#3]
I wonder why no one has ever asked for this. CMake is the preferred build system nowadays.

**Comment by gvanem** [comment]
>  CMake is the preferred build system nowadays.

Yeah right. A confusing pile of mess IMHO. You're welcome to try my collection of GNU-makefiles for it.
It's here: https://www.watt-32.net/misc/Win32plusplus.zip
